values.yaml

```diff
@@ -129,7 +129,7 @@ headless:
 
 # ui service
 service:
-  type: LoadBalancer
+  type: ClusterIP
   httpPort: 8080
   httpsPort: 9443
   nodePort: 30236
```

Comment:
You should remove this or set `nodePort: null` if type is ClusterIP

Reply:
fixed

Reply:
@AyadiAmen what do you think? This seems to be a more standard default for the service type.  

And as a side question: do you reckon changing to ClusterIP by default will impact the FADI userguide or any of our Nifi deployments?

Reply:
@banzo yes it seems to be a more standard default for the service type, and yes it might impact the FADI userguide for instance if the service type is ClusterIP  we can't access the UI with minikube through the command `minikube service -n fadi fadi-nifi` so some updates might be needed. And for the deployments i don't think they're going to be impacted.

Reply:
ok, thanks, seems we use [NodePort](https://github.com/cetic/helm-fadi/blob/master/values.yaml#L264) by default in FADI